#42の対応

mumian1014 opened the pull request:
# 該当のIssue
https://github.com/java-women/techbookfest_2025/issues/42

mumian1014 committed:
Initial Commit

AyaEbata committed:
カート画面を追加

AyaEbata committed:
スタイル修正

AyaEbata committed:
カートに追加するときに重複を避けるようにした

mumian1014 committed:
CartControllerのテスト追加

AyaEbata committed:
ゴミを削除

mumian1014 committed:
単体テスト修正中

mumian1014 committed:
Merge pull request #16 from java-women/cart

1.5.3: Thymeleaf_カート画面

mumian1014 committed:
Merge remote-tracking branch 'origin/feature/#42' into feature/#42

mumian1014 committed:
画面の追加と修正

* 注文完了画面追加
* OrderControllerとOrderServiceのつなぎの修正
* 注文完了画面で注文IDが必要だったため、OrderServiceで注文IDを返却するように修正
* カート画面の修正
    - カートの中がない場合、画面自体が表示されない件の対処
    - カートの中が0円の場合、合計金額をださずにカートが空である旨を出力する対処
* テストクラスのimport文を完全修飾に修正（アスタリスクを使うのをやめた）

AyaEbata reviewed (commented):
本当に細かいところまでありがとうございます！！！！！！！
嬉しくてあやも細かいところまでコメントしちゃいましたてへぺろ

mumian1014 committed:
指摘事項の反映

* OrderService
    - varを使っていた箇所があったので是正しました。
    - 返却値を注文番号ではなく、注文そのものを返却するようにしました。
* Orderが持つ商品はProductではなく、OrderProductに変更しました。
* OrderController
    - varを使っていた箇所があったので是正しました。
    - 返却値の変更に伴う修正をしました。
* 注文完了画面
    - レイアウトを修正しました。
    - 注文日時、合計も表示するようにしました。（OrderはList<OrderProduct>を持ってはいますが、画面が冗長になるので出力はしていません。）
* カート画面
    - EL式の冗長なところをやめました。
    - セッションが空の場合の処理を是正しました。

AyaEbata reviewed (commented):
すみませんーーーーーーーー！
コメント残しましたあああ

mumian1014 committed:
指摘事項の反映

* OrderServiceのフォーマットがおかしかったので是正しました。
* オーダー完了画面のスタイルを見直しました。
* CartControllerの不備を是正しました。
* カート画面のスタイルを修正しました。（不要なクラスの削除）

AyaEbata reviewed (commented):
何度も本当にありがとうございます！！！！！！！！！

mumian1014 committed:
指摘事項の反映

* orderのcssを是正しました。(24pxにしました）
* 注文完了画面の名前を是正しました。(orderだと分かりづらいのでcompleteにしました）

mumian1014 committed:
指摘事項の反映

* cssがorderのままになっていたので、completeに是正しました。

AyaEbata reviewed (approved):
良さそうです！！！！！！
さいこう！！！！！
修正ありがとうございましたッッッッッッッッッ
